conda/recipes/ucxx/recipe.yaml

```diff
@@ -0,0 +1,347 @@
+
+  requirements:
+    build:
+      - ${{ compiler("c") }}
+      - ${{ compiler("cxx") }}
+      - ${{ compiler("cuda") }}
+      - cmake ${{ cmake_version }}
+      - cuda-version =${{ cuda_version }}
+      - ninja
+      - ${{ stdlib("c") }}
+    host:
+      - cuda-version =${{ cuda_version }}
+      - librmm ${{ rapids_version }}
+      - python =${{ py_version }}
+      - rapids-build-backend >=0.3.0,<0.4.0.dev0
+      - ucx
+      - if: cuda_major != "11"
+        then: cuda-cudart-dev
+
+outputs:
+  - package:
+      name: libucxx
+      version: ${{ version }}
+    build:
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+      script:
+        content: |
+          cmake --install cpp/build
+          cmake --install cpp/build --component benchmarks
+      dynamic_linking:
+        overlinking_behavior: "error"
+      prefix_detection:
+        ignore:
+          - lib/libucxx.so
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+        - ${{ stdlib("c") }}
+      host:
+        - cuda-version =${{ cuda_version }}
+        - librmm
+        - ucx
+        - if: cuda_major == "11"
+          then: cudatoolkit
+      run:
+        - if: cuda_major == "11"
+          then: cudatoolkit
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - ${{ pin_compatible("librmm", upper_bound="x.x") }}
+        - ucx >=1.15.0,<1.19.0
+      run_exports:
+        - ${{ pin_subpackage("libucxx", upper_bound="x.x") }}
+      ignore_run_exports:
+        by_name:
+          - cuda-cudart
+          - cuda-version
+          - librmm
+          - python_abi
+          - ucx
+          - if: cuda_major == "11"
+            then: cudatoolkit
+    tests:
+      - package_contents:
+          files:
+            - include/ucxx/address.h
+            - include/ucxx/api.h
+            - include/ucxx/buffer.h
+            - include/ucxx/component.h
+            - include/ucxx/config.h
+            - include/ucxx/constructors.h
+            - include/ucxx/context.h
+            - include/ucxx/delayed_submission.h
+            - include/ucxx/endpoint.h
+            - include/ucxx/exception.h
+            - include/ucxx/future.h
+            - include/ucxx/header.h
+            - include/ucxx/inflight_requests.h
+            - include/ucxx/listener.h
+            - include/ucxx/log.h
+            - include/ucxx/notifier.h
+            - include/ucxx/request.h
+            - include/ucxx/request_helper.h
+            - include/ucxx/request_stream.h
+            - include/ucxx/request_tag.h
+            - include/ucxx/request_tag_multi.h
+            - include/ucxx/typedefs.h
+            - include/ucxx/utils/file_descriptor.h
+            - include/ucxx/utils/python.h
+            - include/ucxx/utils/sockaddr.h
+            - include/ucxx/utils/ucx.h
+            - include/ucxx/worker.h
+            - include/ucxx/worker_progress_thread.h
+            - lib/libucxx.so
```

Comment:
Is it possible to check that some files were _NOT_ included? If so I think it would be useful here to ensure `libucxx_python.so` is not in this package and only in the `ucxx` package.